src/gui_common/ChemicalEquation.cs

```diff
@@ -85,6 +85,12 @@ public bool ShowSpinner
         }
     }
 
+    public bool ShowToggle
+    {
+        get => toggleProcess!.Visible;
+        set => toggleProcess!.Visible = value;
+    }
```

Comment:
Also would it be more efficient if when this is false this node doesn't need to _Process to update the animation?

Reply:
Spinner still spins slightly (and slows down) after disabling the process until it stops completely which looks quite nice. If I do as you say we might lose it.

Reply:
My comment was about the ShowSpinner property (I think I may have commented a line early or a bit late here). So basically my idea: if spinner is not shown at all, it would save on performance if the spinner wasn't attempted to be updated when it isn't visible at all.

Reply:
I see. I don't know what I saw earlier but now that I checked again, this isn't a property you added and there's already code to skip the animation. This comment is resolved.